README.md

```diff
@@ -0,0 +1,128 @@
+
+For the purposes of certfication, the operator application image is accompanied by a metadata bundle image. Both images have to undergo certification tests in RedHat Partner Connect registry, maintainers should be able to obtain access to this registry.
+
+Image definitions for the operator application and the metadata bundle can be found under [`operator/`](operator) [`bundles/`](bundles). Images are mapped to Cilium releases.
+
+Images are built using [GitHub Actions](.github/workflows/ci.yaml) and pushed to [quay.io/cilium/cilium-olm](http://quay.io/cilium/cilium-olm) and [quay.io/cilium/cilium-olm-metadata](http://quay.io/cilium/cilium-olm-metadata). [`imagine`](https://github.com/errordeveloper/imagine) utility is being used to drive the builds.
+
+The metadata bundle image contains just YAML files and no software as such, however it's a crucial part of OLM and when it comes to certification it's what is being tested the most. It's possible to install the operator based on this image, however it's not something that is recommended for Cilium, as Cilium should be installed during cluster bootstrap as otherwise users will have to carry out live network migration.
+
+## Useful Links
+
+- [RedHat Partner Connect](https://connect.redhat.com) (maintainers will be able to obtain access)
+- [operator-courier](https://github.com/operator-framework/operator-courier) - tool used in certification tests
+- [operator-test-playbooks](https://github.com/redhat-operator-ecosystem/operator-test-playbooks) - ansible playbooks used to drive certication test (see [#37](https://github.com/cilium/cilium-olm/issues/37))
+- [Cilium OLM (operator) in RedHat Catalog](https://catalog.redhat.com/software/containers/isovalent/cilium-olm/5ff7310e293738682042b1dd)
+- [Cilium OLM (bundle) in RedHat Catalog](https://catalog.redhat.com/software/containers/isovalent/cilium-olm-metadata/603fd17f69aea331dde395e4)
+
+## Common Workflows
+
+To work with this repository, a maintainer should have Docker with latest version of [`buildx` plugin](https://github.com/docker/buildx) installed, as well as Go toolchain.
+
+To install `kg`, `imagine` and `opm` utilities to `GOPATH`, run:
+
+```
+(cd tools ; go install github.com/errordeveloper/imagine github.com/errordeveloper/kue/cmd/kg github.com/operator-framework/operator-registry/cmd/opm)
+```
+
+### Adding Cilium release
+
+All releases and release candidates should be added to this repo for testing, albeit only latest stable release should be published in the RedHat Catalog.
+
+To add a Cilium release, run:
+
+```
+scripts/add-release.sh 1.10.0
+```
+
+This will do the following:
+
+- create new dirs `{operator,manifests,bundles}/cilium.v1.10.0` populate these with generated configs
+- download Helm chart tarball and unpack it to 
+- create a local commit that has all the changes that can be pushed to the repo
+
+Once the new commit is pushed, new images will be built in GitHub Actions.
```

Comment:
I was trying to drive for instructing the reader not to do, rather than implying they must do something and then describing what happens when they do. Hence the "Push the commit". Otherwise it can be confusing whether the script above is responsible or somehow something else takes responsibility. Direct instructions mean it's very clear that the reader must do this thing and then something will happen.

Reply:
I see, thanks for that! I've noticed previously that automated commits can be confusing to people in general.  Being extra explicit is always a good idea. Hopefully https://github.com/cilium/cilium-olm/commit/5656dad5fb4eaeaa51e88201997887935f4c2198 helps.